src/odemis/gui/comp/overlay/shapes.py

```diff
@@ -20,13 +20,18 @@
 
 """
 from abc import ABCMeta, abstractmethod
-from typing import Tuple, List, Union
+from collections import deque
+from enum import IntEnum
+import logging
+from typing import Tuple, List, Union, Dict, Any
 
 import wx
 
 from odemis import model, util
 from odemis.gui.comp.overlay.base import WorldOverlay, Vec
 
+UNDO_STACK_DEPTH = 25
```

Comment:
can you add a short comment to explain what this value is? Something like: `the number of undo actions stored in the stack`